src/asm/rgbasm.5

```diff
@@ -1598,6 +1633,7 @@ Everything between
 and the matching
 .Ic ENDR
 will be repeated for each value of a given symbol.
+String symbols are not expanded within the symbol name.
```

Comment:
In general, I think we should say that they are not *implicitly* expanded, since `{SYM}` does work. Maybe the documentation for `EQUS` expansion should also mention the word "explicitly", to prime this explanation.

Reply:
I had taken "string symbols are expanded" to always be talking about `EQUS` expansions. `{This}` is consistently called "interpolation", even outside string literals.

Reply:
(This also changes a mention of "`EQUS` symbols" to "string symbols", consistent with other mentions, and to go with the definition: "`EQUS` is used to define string symbols." So just searching for "string symbol" if it's not clear will turn up what this is talking about.)

Reply:
I think it's better to have a little redundancy to reinforce the difference between the two, to further help prevent a reader skimming the document from conflating the two. I don't think that it hurts readability, but I'm no writer.

Reply:
To avoid repeating the same info everywhere, how about saying in the explanation of `{interpolation}` that it will occur "even in contexts where string symbols are not expanded, like `PURGE name`, `DEF(name)`, or `MACRO name`"?

Reply:
Also I can make the term consistently "string equate" instead.

Reply:
Yes please to both!

Reply:
Done :)